src/nvim/charset.c

```diff
@@ -69,28 +69,28 @@ static int chartab_initialized = FALSE;
 /// an error, OK otherwise.
 int init_chartab(void)
 {
-  return buf_init_chartab(curbuf, TRUE);
+  return buf_init_chartab(curbuf, true);
 }
 
 /// Helper for init_chartab
 ///
-/// @param global FALSE: only set buf->b_chartab[]
+/// @param global false: only set buf->b_chartab[]
 ///
 /// @return FAIL if 'iskeyword', 'isident', 'isfname' or 'isprint' option has
 /// an error, OK otherwise.
 int buf_init_chartab(buf_T *buf, int global)
 {
   int c;
-  int c2;
+  bool c2;
```

Comment:
It doesn't seem like `c2` is bool. It gets `-1` as a value and is compared with `int`s at some point.